package.json

```diff
@@ -10,30 +10,17 @@
   "author": "",
   "license": "ISC",
   "private": true,
-  "dependencies": {
-    "@babel/core": "^7.22.1",
-    "@babel/polyfill": "^7.12.1",
-    "@babel/preset-env": "^7.22.4",
-    "autoprefixer": "^10.4.16",
-    "babel-loader": "^9.1.2",
-    "cnbuilder": "^3.1.0",
-    "css-loader": "^6.8.1",
-    "decanter": "^6.2.14",
-    "drupal-behaviors-loader": "^1.0.1",
-    "element-closest-polyfill": "^1.0.6",
-    "element-matches-polyfill": "^1.0.0",
-    "element-qsa-scope": "^1.1.0",
-    "es6-object-assign": "^1.1.0",
-    "filemanager-webpack-plugin": "^8.0.0",
-    "mini-css-extract-plugin": "^2.7.6",
-    "node-sass": "^9.0.0",
-    "optimize-css-assets-webpack-plugin": "^6.0.1",
-    "postcss-loader": "^7.3.2",
-    "postcss-preset-env": "^9.1.4",
-    "sass-loader": "^13.3.1",
-    "style-loader": "^3.3.3",
-    "webpack": "^5.85.0",
-    "webpack-cli": "^5.1.1",
-    "webpack-fix-style-only-entries": "^0.6.1"
+  "devDependencies": {
+    "autoprefixer": "^10.4.18",
+    "browserlist": "^1.0.1",
+    "css-loader": "^6.10.0",
+    "decanter": "^6.3.3",
+    "ignore-emit-webpack-plugin": "^2.0.6",
+    "mini-css-extract-plugin": "^2.8.1",
+    "sass": "^1.72.0",
+    "sass-loader": "^14.1.1",
+    "style-loader": "^3.3.4",
+    "webpack": "^5.91.0",
+    "webpack-cli": "^5.1.4"
```

Comment:
Re-did the webpack compile to just render out sass. Kept it simple and got rid of the other stuff we're not using.